redfish-core/lib/account_service.hpp

```diff
@@ -994,6 +996,82 @@ inline void
         std::variant<std::string>(userNameAttribute));
 }
 
+// This file holds the value of the AllowUnauthACFUpload property.  An
+// Administrator user can GET and PATCH this property.  Contrast with the
+// readonly Oem.IBM.ACFWindowActive property (in /redfish/v1) which is true
+// when either AllowUnauthACFUpload or the "OpPanel function 74 time window"
+// is active.  Low-level design:
+//  - When this file exists, AllowUnauthACFUpload=True.
+//  - Otherwise, AllowUnauthACFUpload=False (the default value).
+const std::filesystem::path allowUnauthACFUploadFilename{
+    "/var/lib/AllowUnauthACFUpload"};
```

Comment:
This gets blasted on a factory reset but would serve a code update/reboot?

Reply:
Correct.  That is the intention.
Note this code is re-written in the next commit to use a new D-Bus property.  That also survives BMC reboot but is reset to default via BMC factory reset.